### PR TITLE
chore(ui-watt): change label color from primary-dark -> neutral-black

### PR DESCRIPTION
**Pull request by Lubbe91**
## Description

change label color to align with Figma

## References

- [#1268](https://app.zenhub.com/workspaces/bounty-hunters-61941fea47ab810010e9aa91/issues/gh/energinet-datahub/greenforce-frontend/1268)

### CODE REVIEW

File: libs/ui-watt/src/lib/foundations/_variables.scss [hunk 1]
```diff
@@ -81,7 +81,7 @@
   --watt-typography-headline-color: var(--watt-color-primary-dark);
   --watt-typography-text-color: var(--watt-on-light-high-emphasis);
   --watt-typography-button-color: var(--watt-color-primary);
-  --watt-typography-label-color: var(--watt-color-primary-dark);
+  --watt-typography-label-color: var(--watt-color-neutral-black);
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/foundations/_variables.scss:84]: question: should it not be `--watt-on-light-high-emphasis`?

image

Comment (Lubbe91) [reply libs/ui-watt/src/lib/foundations/_variables.scss:84]: yes it should 🤦‍♂️

Comment (dzhavat) [reply libs/ui-watt/src/lib/foundations/_variables.scss]: Oh, sorry for being annoying but just noticed that the value of `--watt-on-light-high-emphasis` has changed. Can you adjust it too?

Comment (Lubbe91) [reply libs/ui-watt/src/lib/foundations/_variables.scss]: on it 😎

**Review by dzhavat — APPROVED** [review]

**Pull request merged**